extension-manifest-v2/src/classes/Common.js

```diff
@@ -84,7 +85,7 @@ common.load = async () => {
 
 	setPref('modules.anolysis.enabled', false);
 
-	if (!IS_ANDROID) {
+	if (!IS_ANDROID && HAS_WASM) {
```

Comment:
Found a second place where we check for it:

https://github.com/ghostery/ghostery-extension/blob/33452f0bfba750678fabae0630c870c81aa2b848/extension-manifest-v2/src/classes/Common.js#L128

```
export const setWhotracksmeState = async (enabled) => {
	if (!IS_ANDROID) {
		await setModuleState('hpnv2', enabled);
		await setModuleState('human-web', enabled);
	} else {
		await setModuleState('hpn-lite', enabled);
		await setModuleState('human-web-lite', enabled);
	}
};
```

Perhaps we can unify the logic.